pkg/block/block.go

```diff
@@ -126,6 +128,32 @@ func cleanUp(logger log.Logger, bkt objstore.Bucket, id ulid.ULID, err error) er
 	return err
 }
 
+// MarkForDeletion creates a file
+// which stores information about when the block was marked for deletion.
+func MarkForDeletion(ctx context.Context, logger log.Logger, bkt objstore.Bucket, id ulid.ULID) error {
+	compactorMetaExists, err := objstore.Exists(ctx, bkt, path.Join(id.String(), metadata.CompactorMetaFilename))
+	if err != nil {
+		return errors.Wrap(err, fmt.Sprintf("check compactor meta for id %s in bucket", id.String()))
+	}
+	if compactorMetaExists {
+		level.Info(logger).Log("msg", "compactor-meta already exists for block", "id", id.String())
```

Comment:
Not sure if this needs to be logged at all, perhaps at Debug level only? If so, I'd use full filename here, not just first part of it.

Reply:
Agree. I would not log it at all. Not really interesting outcome and will be spammy even for debug.